src/InstrumentationEngine.Lib/Logging.cpp

```diff
@@ -232,4 +232,17 @@ void CLogging::XmlDumpHelper::WriteUlongNode(const WCHAR* name, const ULONG32 va
         << setfill(_T('0')); // fill with 0s
 
     m_result << m_childrenIndent << _T('<') << name << _T('>') << hex << setw(10) << value << _T("</") << name << _T('>') << _T("\r\n");
+}
+
+// Implemenation of Assert functions found in Macros.h
+void AssertLogFailure(_In_ const WCHAR* wszError, ...)
+{
```

Comment:
Should there be a IfNotInitRet(s_initialize); call?

Reply:
GetLoggingFlags will do that.